packages/terra-tabs/src/common-tabs/_Tab.jsx

```diff
@@ -235,6 +288,16 @@ const Tab = ({
         {customDisplay || icon}
         {(!customDisplay && !isIconOnly) && <span className={variant === 'framework' ? cy('label') : cx('label')}>{label}</span>}
       </div>
+      {isClosable && (
+      <div
+        className={cx('tabs-remove-button')}
+        type="button"
+        aria-label={tabDeleteLabel}
+        onClick={onCloseClick}
+      >
+        <IconClose a11yLabel="Close Button" />
```

Comment:
"Close Button" requires a translation !!

Reply:
This Icon as added for decorative icon, which wont get focus hence removing the a11ylabel